source/ftrack_connect/ui/widget/login.py

```diff
@@ -125,6 +128,16 @@ def __init__(self, *args, **kwargs):
         )
         layout.addSpacing(20)
 
+        if self.server_urls:
+            self.server_completer = QtWidgets.QCompleter(
+                list(self.server_urls.keys()), self)
+            self.server_completer.setCompletionMode(
+                QtWidgets.QCompleter.PopupCompletion)
+            self.server.setCompleter(self.server_completer)
+            self.server_completer.activated.connect(self.server_url_chosen)
+            if os.getenv('FTRACK_SERVER', ''):
```

Comment:
@nebukadhezer I'd rather not rely on environment variable, as it might lead to unknown mix of api key and hosts.
Let the config drive the login only , so is predictable.

Reply:
I want when artists laucnhes connect the first time that she/he does not have to enter the domain by hand... it is optional anyway if the envvar is not set, I guess 99% of the time it is empty anyway.
But with sth like rez (or sh,bat scripts )I can create different connect launchers with different domains... and the user just has to login...

One thing I would want to point out in the inital login page too, is that the api key is being resetted.

Reply:
@nebukadhezer 
can you , then, add a warning if the values are picked up from environment variables expressed externally ? 
it would be important for us to know while debugging client issues.